tools/ci_build/build.py

```diff
@@ -358,6 +359,12 @@ def parse_arguments():
     parser.add_argument(
         "--build_micro_benchmarks", action='store_true',
         help="Build ONNXRuntime micro-benchmarks.")
+    parser.add_argument(
```

Comment:
What about opset ver?

Reply:
opset versions are collected from either the model or the file.

Reply:
Is there a need for an opset ver to be specified separately? cc @skottmckay

Reply:
In order to minimize what's included in the build it's better to have op+opset. If multiple models are provided the opsets can be read from those (this applies to the OXO scenario as the two models they're given us have different opsets). for the csv usage I'm not sure what's required. something equivalent to what is processed from the model  would be good.

in my script it was {domain:{opset:[ops]}} but in this version it's {optype:{domain:[opsets]}}. that change is a little counter-intuitive given an op usually only exists in one domain, and for an individual model the opset is per domain

---
In reply to: [468801946](https://github.com/microsoft/onnxruntime/pull/4747#discussion_r468801946) [](ancestors = 468801946)

Reply:
domain:opset:ops is indeed more generic, will refactor this part.

Reply:
Done.